core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2597,6 +2601,28 @@ public void setAttribute(AlluxioURI path, SetAttributeOptions options)
   }
 
   /**
+   * Checks whether the owner belongs to the group.
+   * @param owner The owner to check
+   * @param group The group to check
+   * @throws FailedPreconditionException if owner does not belong to group
+   * @throws AccessControlException if this operation is not permitted
+   */
+  private void checkUserBelongsToGroup(String owner, String group)
+  throws FailedPreconditionException, AccessControlException {
+    try {
+      List<String> groups = null;
+      groups = CommonUtils.getGroups(owner);
```

Comment:
nit: can you combine those to be one line ?
`List<String> groups = CommonUtils.getGroups(owner);`